tests/testthat/_snaps/derive_vars_cat.md

```diff
@@ -0,0 +1,188 @@
+         USUBJID     VSTEST  AVAL AVALCAT1        AVALCA1N
+         <chr>       <chr>  <dbl> <chr>              <dbl>
+       1 01-701-1015 Height 147.  Height < 160           2
+       2 01-701-1023 Height 163.  Height >= 160          1
+       3 01-701-1028 Height 178.  Height >= 160          1
+       4 01-701-1033 Height 175.  Height >= 160          1
+       5 01-701-1034 Height  NA   <NA>                  NA
+       6 01-701-1047 Height  NA   <NA>                  NA
+       7 01-701-1097 Height 169.  Height >= 160          1
+       8 01-701-1111 Height 158.  Height < 160           2
+       9 01-701-1115 Height 182.  Height >= 160          1
+      10 01-701-1118 Height 180.  Height >= 160          1
+      11 01-701-1015 Weight  54.0 Weight < 66.68         2
+      12 01-701-1023 Weight  78.5 Weight >= 66.68        1
+      13 01-701-1028 Weight  98.9 Weight >= 66.68        1
+      14 01-701-1033 Weight  88.4 Weight >= 66.68        1
+      15 01-701-1034 Weight  NA   <NA>                  NA
+      16 01-701-1047 Weight  NA   <NA>                  NA
+      17 01-701-1097 Weight  78.0 Weight >= 66.68        1
+      18 01-701-1111 Weight  60.3 Weight < 66.68         2
+      19 01-701-1115 Weight  78.7 Weight >= 66.68        1
+      20 01-701-1118 Weight  71.7 Weight >= 66.68        1
+
+# derive_vars_cat Test 10: Adding an extra variable (flag) to the dataset
+
+    Code
+      result
+    Output
+      # A tibble: 20 x 6
+         USUBJID     VSTEST  AVAL AVALCAT1 AVALCA1N extra_var
+         <chr>       <chr>  <dbl> <chr>       <dbl> <lgl>    
+       1 01-701-1015 Height 147.  <160            2 FALSE    
+       2 01-701-1023 Height 163.  >=160           1 TRUE     
+       3 01-701-1028 Height 178.  >=160           1 TRUE     
+       4 01-701-1033 Height 175.  >=160           1 TRUE     
+       5 01-701-1034 Height  NA   <NA>           NA NA       
+       6 01-701-1047 Height  NA   <NA>           NA NA       
+       7 01-701-1097 Height 169.  >=160           1 TRUE     
+       8 01-701-1111 Height 158.  <160            2 FALSE    
+       9 01-701-1115 Height 182.  >=160           1 TRUE     
+      10 01-701-1118 Height 180.  >=160           1 TRUE     
+      11 01-701-1015 Weight  54.0 <NA>           NA NA       
+      12 01-701-1023 Weight  78.5 <NA>           NA NA       
+      13 01-701-1028 Weight  98.9 <NA>           NA NA       
+      14 01-701-1033 Weight  88.4 <NA>           NA NA       
+      15 01-701-1034 Weight  NA   <NA>           NA NA       
+      16 01-701-1047 Weight  NA   <NA>           NA NA       
+      17 01-701-1097 Weight  78.0 <NA>           NA NA       
+      18 01-701-1111 Weight  60.3 <NA>           NA NA       
+      19 01-701-1115 Weight  78.7 <NA>           NA NA       
+      20 01-701-1118 Weight  71.7 <NA>           NA NA       
+
+# derive_vars_cat Test 12: definition has wrong shape
+
+    Failed to convert `definition` to tribble: Data must be rectangular.
```

Comment:
I believe tribble is the function used to create tibbles? https://tibble.tidyverse.org/reference/tribble.html

Should this be 
```suggestion
    Failed to convert `definition` to tibble: Data must be rectangular.
```

But does a user ever really do something with a tibble()?  Should we say something about `exprs()` in here as well as it could be confusing to user?  I guess I am finding it confusing.

Reply:
I tried to explain this better.